ci/test/wrap-wine.sh

```diff
@@ -13,7 +13,13 @@ for b_name in {"${BASE_OUTDIR}/bin"/*,src/secp256k1/*tests,src/minisketch/test{,
         echo "Wrap $b ..."
         mv "$b" "${b}_orig"
         echo '#!/usr/bin/env bash' > "$b"
-        echo "( wine \"${b}_orig\" \"\$@\" ) || ( sleep 1 && wine \"${b}_orig\" \"\$@\" )" >> "$b"
+        if test "$(basename "$b")" = "test_bitcoin.exe"; then
```

Comment:
any need for this condition?

Reply:
Yes. Without it python tests in test/util got crazy. Did not investigate further though.

Reply:
> Probably it would be more ideal to fix this by using _script_ in `src/Makefile.test.include` instead of the less reliable `> $<.log 2>&1 || (cat $<.log && false)` which could cause buffered output to be lost on other platforms, not just windows. If _script_ were used there, the changes here could be reverted.

Great idea, but BSD version of `script` on macOS is incompatible with Linux version, e.g., it lacks the `--return` option.

Reply:
> Great idea, but BSD version of script on macOS is incompatible with Linux version, e.g., it lacks the --return option.

Candidate for python port? :sweat_smile: